exercises/nucleotide-count/HINTS.md

```diff
@@ -0,0 +1,3 @@
+##HINTS
+For this exercise the following C# feature comes in handy: 
```

Comment:
Actually, this exercise requires the use of a Dictionary, so I would rephrase:

"This exercise requires you to return a dictionary. For more information, see [this page](https://msdn.microsoft.com/en-us/library/s4ys34ea(v=vs.110).aspx)."